packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -347,14 +347,24 @@ Outputs:
     Description: The domain name of the S3 bucket owned by the CDK toolkit stack
     Value:
       Fn::Sub: "${StagingBucket.RegionalDomainName}"
+  FileAssetKeyArn:
```

Comment:
Do we want to add this to some integ test?

Reply:
How about we add unit tests on the narrow contract of the bootstrap stack?

Reply:
Sounds good.. Anything that will protect us from changing it accidentally.